.github/workflows/production-suite.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: /srv/zulip-venv-cache
-          key: v1-venv-${{ github.job }}-${{ hashFiles('requirements/thumbor-dev.txt') }}-${{ hashFiles('requirements/dev.txt') }}
+          key: v1-venv-${{ github.job }}-${{ hashFiles('requirements/dev.txt') }}
```

Comment:
Should `prod.txt` be in these hashes?

Reply:
No, the next step is to provision using `dev.txt` so we can build the release tarball.

Reply:
Ahh, this is that stage, great.